pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1050,6 +1050,10 @@ class Build {
                 metaWrittenOut = true
             }
 
+            // Special handling for sbom metadat file (to be backwards compatible for api service)
+            if (file.contains("sbom")) {
+                file.replace(".json", "-metadata") // from "*sbom<XXX>.json" to "*sbom<XXX>-metadata"
```

Comment:
not sure you want to inline replace file variable, I would put the context.writeFile line in both sides of an if (file.contains("sbom")) {context.writeFile file:file.replace(".json", "-metadata") .....} else {..as before.}